src/app/services/api/api.service.ts

```diff
@@ -98,38 +158,57 @@ export class ApiService {
    * Gets information about the current coin supply.
    */
   getCoinSupply(): Observable<GetCoinSupplyResponse> {
-    return this.get('coinSupply');
+    const url = !this.nodeUrl() ? 'coinSupply' : 'v1/coinSupply';
+
+    return this.get(url);
   }
 
   /**
    * Gets the list of unspent outputs of an address.
    * @param address Address to consult.
    */
   getCurrentBalance(address: string): Observable<GetCurrentBalanceResponse> {
-    return this.get('currentBalance', { addrs: address });
+    const url = !this.nodeUrl() ? 'currentBalance' : 'v1/outputs';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin-explorer/pullRequest?prid=7163626)
```suggestion
    const url = !this.nodeUrl() ? "currentBalance" : 'v1/outputs';
```